client/Elements/Components/CreateTopicAdvanceWizard/CreateTopicAdvanceWizard.patternfly.css

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+ 
+.topics-wizard-content {
+  max-width: 500px;
+ }
```

Comment:
This is in 2 PRs now (this and https://github.com/Redhat-AppServices/strimzi-ui/pull/8). We should have a common file that defines a var for the max width used for both.